guides/common/attributes-satellite.adoc

```diff
@@ -63,6 +63,7 @@
 :customssl: custom SSL
 :DocState: satellite
 :EL: {RHEL}
+:EL-abbr: {RHEL}
```

Comment:
```suggestion
:EL-abbr: RHEL
```
The point is to use the actual abbreviation, so let's lose the brackets.

Reply:
Same in guides/common/attributes-base.adoc